contract/tests/test_contract.py

```diff
@@ -2456,4 +2456,7 @@ def test_automatic_price_with_specific_uom(self):
 
     def test_check_month_name_marker(self):
         invoice_id = self.contract3.recurring_create_invoice()
-        self.assertEqual(invoice_id.invoice_line_ids[0].name, "Header for May Services")
```

Comment:
@damdam-s Because we are in June :smile:

Reply:
I fix this in this PR https://github.com/OCA/contract/pull/961.
in fact date set on contract level are not taken into account https://github.com/OCA/contract/blob/14.0/contract/models/contract.py#L168